docs/mmteb/points/539.jsonl

```diff
@@ -0,0 +1,2 @@
+{"GitHub": "dipam7", "New dataset": 2}
```

Comment:
@imenelydiaker would this be 6 points? (4 for new language and 2 for dataset)

Reply:
It should be 2, because we already handle Spanish in one of the multilingual tasks, see [AmazonReviewsClassification](https://github.com/embeddings-benchmark/mteb/blob/main/mteb/tasks/Classification/multilingual/AmazonReviewsClassification.py)

Reply:
@imenelydiaker any reason why this one didn't auto merge after you enabled it?

Reply:
I updated the branch & tests will run, it should be good this time

Reply:
One test is failing, can you please run `make pr` locally, fix issues then push them?